fearless/Modules/Staking/StakingAmount/StakingAmountInteractor.swift

```diff
@@ -92,16 +92,20 @@ extension StakingAmountInteractor: StakingAmountInteractorInputProtocol, Runtime
 
         provideRewardCalculator()
 
-        fetchConstant(
-            for: .existentialDeposit,
-            runtimeCodingService: runtimeService,
-            operationManager: operationManager
-        ) { [weak self] (result: Result<BigUInt, Error>) in
-            switch result {
-            case let .success(amount):
-                self?.presenter.didReceive(minimalBalance: amount)
-            case let .failure(error):
-                self?.presenter.didReceive(error: error)
+        if chain.isOrml {
+            presenter.didReceive(minimalBalance: BigUInt.zero)
```

Comment:
`0` not being accepted? I believe `BigUInt` conforms to ExpressedByIntegerLiteral